### PR TITLE
Reparo: add safe-mode in reapro example config file 

### PR DESCRIPTION
**Pull request by ericsyh**
### What problem does this PR solve? 
WX added the safe-mode config in reparo with https://github.com/pingcap/tidb-binlog/pull/652, this PR updates the reparo example config file. 

### What is changed and how it works?
Add the safe-mode config in /cmd/reparo/reparo.toml

### CODE REVIEW

File: cmd/reparo/reparo.toml [hunk 1]
```diff
@@ -19,25 +19,29 @@ log-level = "info"
...
 #[[replicate-ignore-table]]
-#db-name ="test"
+#db-name = "test"
 #tbl-name = "~^a.*"
 
 [dest-db]
 host = "127.0.0.1"
 port = 3309
 user = "root"
-password = ""
```
Comment (suzaku) [inline cmd/reparo/reparo.toml]: Keep the new line removed?

Comment (ericsyh) [reply cmd/reparo/reparo.toml]: I checked my local branch, there is no new line in the end.

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**